src/demo/data.js

```diff
@@ -0,0 +1,51 @@
+import faker from 'faker';
+import courses from './courses';
+
+const numEnrollments = 330;
+const percentPassed = 0.45;
+const percentActive = 0.35;
+
+const enrollments = [...Array(numEnrollments)].map((_, idx) => {
+  // Although we don't pass first & lastname, this makes our emails more consistent
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+
+  const course = courses[faker.random.number({ max: courses.length - 1 })];
+  const courseStart = faker.date.past();
+  const courseEnd = faker.date.future(null, courseStart);
+  const hasPassed = Math.random() <= percentPassed;
+  const grade = hasPassed ? faker.random.number({ min: 70, max: 100 }) :
+    faker.random.number({ max: 70 });
+
+  // TODO: we could generate all columns so we could easily fake the csv call as well
```

Comment:
Do we know if the csv download is something sales/product would want to demo? If so, we should probably include it in this PR.

Reply:
Good question - I talked to product and they feel opening the downloaded CSV would not be part of the demo so I left it off for now. Could always add that in later if we/they find it useful.